master/docs/developer/database.rst

```diff
@@ -550,6 +550,16 @@ logs
         It should only be called for finished logs.
         This method may take some time to complete.
 
+    .. py:method:: deleteOldLogChunks(older_than_timestamp)
+
+        :param integer older_than_timestamp: the logs whose step's ``started_at`` is older than ``older_than_timestamp`` will be deleted.
```

Comment:
I was trying to simplify the sql request. i don't think this corner case is worth the additional join. We can make it clear in the UI why this step log was deleted